pyrk/data/tests/test_precursors.py

```diff
@@ -1,5 +1,2 @@
-from nose.tools import assert_false
-
-
 def test_stuff():
-    assert_false(False)
+    assert not False
```

Comment:
Interesting. I realize now this is an empty placeholder where precursor tests should be. No edit needed... just a comment for myself...